src/Bicep.Core/Workspaces/SourceFileGroupingBuilder.cs

```diff
@@ -177,6 +177,15 @@ private void PopulateRecursive(BicepSourceFile file, IFeatureProviderFactory fea
                 }
                 uriResultByArtifactReferenceSyntaxLookup[restorable] = uriResult;
 
+                if (childArtifactReference is { }) {
```

Comment:
uriResultByBicepSourceFileByArtifactReference had been uriResultByBicepSourceFileByArtifactReference but not implemented.

Reply:
I tried spending some time pondering on this but I couldn't understand what this comment aims to convey, could you please providing some more context?

Reply:
Doesn't this add values to it? 
https://github.com/Azure/bicep/blob/sw/f12-nested-links4/src/Bicep.Core/Workspaces/SourceFileGroupingBuilder.cs#L184-L185